docs/preview/features/powershell/azure-api-management.md

```diff
@@ -45,32 +45,48 @@ PS> Create-AzApiManagementApiOperation -ResourceGroup $ResourceGroup -ServiceNam
 # New API operation '$OperationName' on API Management service was added.
 ```	
```

Comment:
This looks like a doc bug or not?
```suggestion
```

Reply:
No, it's the powershell comment that is here presented as the output of the command 😃 .